### PR TITLE
Update ir.m3u

### PR DESCRIPTION
**Pull request by mh292929**
i update ir.m3u with irib correct link

### CODE REVIEW

**Comment by MH140000** [comment]
all stream URL starts with http://185.236.230.222 are broken, i change these channel to another server.

**Review by MapGuy11 — APPROVED** [review]
Looks Good!

File: channels/ir.m3u [hunk 1]
```diff
@@ -79,13 +79,12 @@ https://live1.presstv.com/live/hispan.m3u8
 https://live.presstv.com/live/smil:live.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="HodHodTV.ir" tvg-name="HodHod TV" tvg-country="IR" tvg-language="" tvg-logo="https://i.imgur.com/TMxCgX1.jpg" group-title="",HodHod TV
 http://51.210.199.12/hls/stream.m3u8
-#EXTINF:-1 tvg-id="iFILM.ir" tvg-name="iFILM" tvg-country="IR" tvg-language="Persian" tvg-logo="" group-title="",iFILM
-http://185.236.230.222:8000/play/a01n
-#EXTINF:-1 tvg-id="iFILMArabic.ir" tvg-name="iFILM Arabic" tvg-country="IR" tvg-language="Arabic" tvg-logo="https://i.imgur.com/RPwKZUD.jpg" group-title="Movies",iFILM Arabic (576p)
-http://163.172.209.107:8000/play/a04j/index.m3u8
+
```
Comment (LaneSh4d0w) [inline channels/ir.m3u:82]: Take off that new line.

File: channels/ir.m3u [hunk 2]
```diff
@@ -79,13 +79,12 @@ https://live1.presstv.com/live/hispan.m3u8
 https://live.presstv.com/live/smil:live.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="HodHodTV.ir" tvg-name="HodHod TV" tvg-country="IR" tvg-language="" tvg-logo="https://i.imgur.com/TMxCgX1.jpg" group-title="",HodHod TV
 http://51.210.199.12/hls/stream.m3u8
-#EXTINF:-1 tvg-id="iFILM.ir" tvg-name="iFILM" tvg-country="IR" tvg-language="Persian" tvg-logo="" group-title="",iFILM
-http://185.236.230.222:8000/play/a01n
-#EXTINF:-1 tvg-id="iFILMArabic.ir" tvg-name="iFILM Arabic" tvg-country="IR" tvg-language="Arabic" tvg-logo="https://i.imgur.com/RPwKZUD.jpg" group-title="Movies",iFILM Arabic (576p)
-http://163.172.209.107:8000/play/a04j/index.m3u8
```
Comment (LaneSh4d0w) [inline channels/ir.m3u:85]: This link works.
vlcsnap-2021-07-23-17h12m52s953

File: channels/ir.m3u [hunk 3]
```diff
@@ -100,93 +99,91 @@ http://51.210.199.57/hls/stream.m3u8
...
+https://sdm.telewebion.com/live/bushehr/playlist.m3u8
 #EXTINF:-1 tvg-id="IRIBDENA.ir" tvg-name="IRIB DENA" tvg-country="IR" tvg-language="Persian" tvg-logo="" group-title="",IRIB DENA
-http://185.236.230.222:8000/play/a00v
+https://sdm.telewebion.com/live/dena/playlist.m3u8
 #EXTINF:-1 tvg-id="IRIBESFAHAN.ir" tvg-name="IRIB ESFAHAN" tvg-country="IR" tvg-language="Persian" tvg-logo="" group-title="",IRIB ESFAHAN
-http://185.236.230.222:8000/play/a00d
+https://sdm.telewebion.com/live/esfahan/playlist.m3u8
 #EXTINF:-1 tvg-id="IRIBFARS.ir" tvg-name="IRIB FARS" tvg-country="IR" tvg-language="Persian" tvg-logo="" group-title="",IRIB FARS
-http://185.236.230.222:8000/play/a00t
+http://cdn1.live.irib.ir:1935/epg-live/smil:fars/chunklist_b307200_slFa.m3u8
```
Comment (LaneSh4d0w) [inline channels/ir.m3u:116]:
```suggestion
http://cdn1.live.irib.ir:1935/epg-live/smil:fars/playlist.m3u8
```

File: channels/ir.m3u [hunk 4]
```diff
@@ -100,93 +99,91 @@ http://51.210.199.57/hls/stream.m3u8
...
+https://sdm.telewebion.com/live/nasim/playlist.m3u8
 #EXTINF:-1 tvg-id="IRIBOFOGH.ir" tvg-name="IRIB OFOGH" tvg-country="IR" tvg-language="Persian" tvg-logo="" group-title="",IRIB OFOGH
-http://185.236.230.222:8000/play/a01m
+https://sdm.telewebion.com/live/ofogh/playlist.m3u8
 #EXTINF:-1 tvg-id="IRIBQURAN.ir" tvg-name="IRIB QURAN" tvg-country="IR" tvg-language="Persian" tvg-logo="" group-title="",IRIB QURAN
-http://185.236.230.222:8000/play/a01h
+https://sdm.telewebion.com/live/quran/playlist.m3u8
 #EXTINF:-1 tvg-id="IRIBSAHAND.ir" tvg-name="IRIB SAHAND" tvg-country="IR" tvg-language="Persian" tvg-logo="" group-title="",IRIB SAHAND
-http://185.236.230.222:8000/play/a00z
+http://cdn1.live.irib.ir:1935/epg-live/smil:sahand/chunklist_b307200_slFa.m3u8
```
Comment (LaneSh4d0w) [inline channels/ir.m3u:132]:
```suggestion
http://cdn1.live.irib.ir:1935/epg-live/smil:sahand/playlist.m3u8
```

File: channels/ir.m3u [hunk 5]
```diff
@@ -100,93 +99,91 @@ http://51.210.199.57/hls/stream.m3u8
...
+https://sdm.telewebion.com/live/aflak/playlist.m3u8
 #EXTINF:-1 tvg-id="IRIBAFTAB.ir" tvg-name="IRIB-AFTAB" tvg-country="IR" tvg-language="Persian" tvg-logo="" group-title="",IRIB-AFTAB
-http://185.236.230.222:8000/play/a00q
+https://sdm.telewebion.com/live/aftab/playlist.m3u8
 #EXTINF:-1 tvg-id="IRIBBARAN.ir" tvg-name="IRIB-BARAN" tvg-country="IR" tvg-language="Persian" tvg-logo="" group-title="",IRIB-BARAN
-http://185.236.230.222:8000/play/a00m
+https://sdm.telewebion.com/live/baran/playlist.m3u8
 #EXTINF:-1 tvg-id="IRIBGOLESTAN.ir" tvg-name="IRIB-GOLESTAN" tvg-country="IR" tvg-language="Persian" tvg-logo="" group-title="",IRIB-GOLESTAN
-http://185.236.230.222:8000/play/a00i
+http://cdn1.live.irib.ir:1935/epg-live/smil:golestan/chunklist_b307200_slFa.m3u8
```
Comment (LaneSh4d0w) [inline channels/ir.m3u:162]:
```suggestion
http://cdn1.live.irib.ir:1935/epg-live/smil:golestan/playlist.m3u8
```

File: channels/ir.m3u [hunk 6]
```diff
@@ -100,93 +99,91 @@ http://51.210.199.57/hls/stream.m3u8
...
+https://sdm.telewebion.com/live/khoozestan/playlist.m3u8
 #EXTINF:-1 tvg-id="IRIBKORDESTAN.ir" tvg-name="IRIB-KORDESTAN" tvg-country="IR" tvg-language="Persian" tvg-logo="" group-title="",IRIB-KORDESTAN
-http://185.236.230.222:8000/play/a00h
+https://sdm.telewebion.com/live/kordestan/playlist.m3u8
 #EXTINF:-1 tvg-id="IRIBNOOR.ir" tvg-name="IRIB-NOOR" tvg-country="IR" tvg-language="Persian" tvg-logo="" group-title="",IRIB-NOOR
-http://185.236.230.222:8000/play/a00o
+https://sdm.telewebion.com/live/noor/playlist.m3u8
 #EXTINF:-1 tvg-id="IRIBQazvin.ir" tvg-name="IRIB-Qazvin" tvg-country="IR" tvg-language="Persian" tvg-logo="" group-title="",IRIB-Qazvin
-http://185.236.230.222:8000/play/a00n
+http://cdn1.live.irib.ir:1935/epg-live/smil:qazvin/chunklist_b307200_slFa.m3u8
```
Comment (LaneSh4d0w) [inline channels/ir.m3u:184]:
```suggestion
http://cdn1.live.irib.ir:1935/epg-live/smil:qazvin/playlist.m3u8
```

**Comment by LaneSh4d0w** [comment]
Otherwise, looks pretty good for a first PR.

File: channels/ir.m3u (hunk 1 above)
Comment (MH140000) [reply channels/ir.m3u:82]: is it good now?

**Review by BellezaEmporium — APPROVED** [review]

**Comment by LaneSh4d0w** [comment]
Thank you for your contribution !

**Comment by MH140000** [comment]
please add these channels:
Bartar Tv
entertain
http://159.69.58.154/bartar/bartar.m3u8
4U TV
entertain
http://116.202.255.113:1935/4utv/livehd/playlist.m3u8
Film 1
movies
http://159.69.58.154/film1/film1tv.m3u8
Channel One
Political
https://2nbyjjx7y53k-hls-live.5centscdn.com/cls040317/0070c5b7ef083bdc8de09065c61a34d8.sdp/playlist.m3u8
bbc persian s2
news
http://159.69.58.154/bbc/master.m3u8
Iran International
news
https://dev-live.livetvstream.co.uk/LS-63503-4/index.m3u8
Toonix
kids
http://51.210.227.134/hls/stream.m3u8
T2 Movies
movies
http://159.69.58.154/t2_movies/master.m3u8
PMC Royale
music
http://51.210.199.29/hls/stream.m3u8
Gem Series
movies
http://216.66.42.37:7777/GemSeries_HD.m3u8
NAVAHANG
music
http://51.210.227.130/hls/stream.m3u8
FX
movies
http://51.210.199.52/hls/stream.m3u8
Ecran tv
entertain
http://51.210.199.40/hls/stream.m3u8
Irane Aryaee
Political
http://159.69.58.154/irane_arya/irane_arya.m3u8
avang tv
music
http://appavang.flashmediacast.com:1935/Appavang/livestream/playlist.m3u8
Persiana Game&Tech
documentary
http://51.210.199.25/hls/stream.m3u8
Persiana Sonati
music
http://51.210.199.59/hls/stream.m3u8
Ekran Movie
movies
http://159.69.58.154/ekran/ekrantv.m3u8
Persiana Korea
movies
http://51.210.199.14/hls/stream.m3u8
Manoto +1
entertain
http://159.69.58.154/manoto_plus1/manoto_plus.m3u8
Manoto +2
entertain
http://159.69.58.154/manoto_plus2/manoto2.m3u8
TehranTV 24
movies
http://51.210.227.132/hls/stream.m3u8
Persiana Comedy
movies
http://51.210.199.27/hls/stream.m3u8
Tin tv
entertain
https://bozztv.com/1gbw5/tintv/tintv/playlist.m3u8
Health Media
documentary
https://j78dpkrjlq5r-hls-live.5centscdn.com/HMN/271ddf829afeece44d8732757fba1a66.sdp/playlist.m3u8
Azadi tv
Political
https://5b18be6964c2f.streamlock.net/m/azadah/playlist.m3u8
MTC
music
http://mellitv.tulix.tv/mellitv/myStream.sdp/playlist.m3u8
4 Kurd
music
http://51.210.199.28/hls/stream.m3u8
Eshare TV
movies
http://159.69.58.154/eshare/esharetv.m3u8
Bazme Asheghan
movies
https://5dc143936e9ae.streamlock.net/saeed_nl/saeed_nl/chunklist.m3u8
CPMTV 24
documentary
http://159.69.58.154/cpmtv/cpmtv.m3u8
Persian Film
movies
http://51.210.227.135/hls/stream.m3u8
1TvPersian
entertain
https://11778-1.b.cdn13.com/onetvpersia001.m3u8?bandwidth=3196&playlistlength=5&shift=0&v=2
Nour
Religious
https://cdn.videoevent.live:19360/elfaro4/elfaro4.m3u8
Persiana Classic
movies
http://51.210.199.26/hls/stream.m3u8
Persiana Iranian
movies
http://51.210.199.22/hls/stream.m3u8
AMG
movies
https://nl.livekadeh.com/hls2/AMG.m3u8
Faraz TV
movies
https://faraztv.net/hls/stream.m3u8
Music 1
music
http://159.69.58.154/music1/music1_tv.m3u8
High Vision
music
https://streamer1.connectto.com/HIGHVISION_WEB_1205/tracks-v1a1/mono.m3u8
Kidstoons
kids
http://de2.martiatv.com:8080/hls/kidstvlive.m3u8
Parsiland
movies
http://vps.parsiland.net/hls/ParsiLand.m3u8
Payam Javan TV
movies
http://uni01rtmp.tulix.tv/kensecure/pjtv.stream/playlist.m3u8
AFN TV
entertain
https://bozztv.com/1gbw5/tintv2/tintv2/playlist.m3u8
Persian Bazar
other
https://stream.persiantv1.com/ptv1/playlist1/index.m3u8
ITC
entertain
https://dacastmmd.mmdlive.lldns.net/dacastmmd/f05d55e42dc746c8bd36edafbace7cc1/chunklist_b4628000.m3u8
Kalemeh TV
Religious
http://51.210.199.37/hls/stream.m3u8
Icc Tv
movies
http://mn-nl.mncdn.com/ezcdn-nownews/live_1/playlist.m3u8
Negah Tv
entertain
https://iptv.negahtv.com/negahtv/playlist2/index.m3u8
Khatereh Tv
movies
https://5caf24a595d94.streamlock.net:1937/8130/8130/playlist.m3u8
Omide Iran
entertain
http://51.210.199.38/hls/stream.m3u8
Khaneh Honarmandan
movies
http://159.69.58.154/honarmandan/honarmandan.m3u8
Sat7 Pars
Religious
https://svs.itworkscdn.net/sat7parslive/sat7pars.smil/sat7parspublish/sat7pars_720p/chunks_dvr.m3u8
Payame Aramesh
Religious
http://51.15.133.13/hls/stream.m3u8
Salam TV
Religious
https://iptv.salaamtv.org/salaam/playlist.m3u8
Icnet TV
Religious
http://51.210.199.7/hls/stream.m3u8
BeitolAbbas Tv
Religious
http://live.beitolabbas.tv/live/beitolabbastv.m3u8
Imam Hussein TV
Religious
https://fa.imamhusseintv.com/live/ih1.m3u8
Wesal haq
Religious
https://stream.holol.com/w/whaq_360p/playlist.m3u8
marjaeyat
Religious
https://livefa.marjaeyattv.com/mtv_fa/playlist.m3u8
valiasr
Religious
https://ir13.livekadeh.com/hls2/valiasr.m3u8
Sepanj Tv
movies
http://51.210.199.30/hls/stream.m3u8
Persiana Sport
sport
http://159.69.58.154/varzesh3/varzesh3_tv.m3u8
Mono Tv
sport
http://51.210.227.137/hls/stream.m3u8
tapesh 2 s2
music
http://51.210.199.44/hls/stream.m3u8
hasti tv
https://live.hastitv.com/hls/livetv.m3u8

**Pull request merged**